x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/index_data_visualizer_view/index_data_visualizer_view.tsx

```diff
@@ -368,9 +368,16 @@ export const IndexDataVisualizerView: FC<IndexDataVisualizerViewProps> = (dataVi
         earliest,
         latest
       );
+      // Because load overall stats perform queries in batches
+      // there could be multiple errors
+      if (Array.isArray(allStats.errors) && allStats.errors.length > 0) {
+        allStats.errors.forEach((err: any) => {
+          dataLoader.displayError(err.body?.message?.output?.payload ?? err);
```

Comment:
it might be worth testing this error with our `extractErrorProperties` function to see if it displays it extracts the error text correctly, if so, I think we should copy that function over to tis plugin.
The shape of errors returned from es and kibana can depend on where they were thrown from, `extractErrorProperties` does a decent job of recognising all known error shapes.

Reply:
Added the new utility here https://github.com/elastic/kibana/pull/104567/commits/4c2b680f0badbcf3ab907f8aa7082dd895661e54